app/screens/settings-screen/account/profile.tsx

```diff
@@ -0,0 +1,117 @@
+import { ScrollView } from "react-native-gesture-handler"
+import { Screen } from "@app/components/screen"
+import { GaloyPrimaryButton } from "@app/components/atomic/galoy-primary-button"
+import { useI18nContext } from "@app/i18n/i18n-react"
+import { TouchableOpacity, View } from "react-native"
+import { GaloyIcon } from "@app/components/atomic/galoy-icon"
+import { makeStyles, Text } from "@rneui/themed"
+
+export const ProfileScreen: React.FC = () => {
+  const styles = useStyles()
+  const { LL } = useI18nContext()
+
+  const data = [
+    {
+      username: "User 1",
+      selected: true,
+    },
+    {
+      username: "User 2",
+      selected: false,
+    },
+    {
+      username: "User 3",
+      selected: false,
+    },
+  ]
```

Comment:
This data should come in dynamically from the keychain, it's ok to hardcode this during development but we shouldn't merge this code as is. Please replace this with appropriate keychain helper.